test/numa/tests/memtier/n4c16/test01-only-add-pods/code.var.sh

```diff
@@ -0,0 +1,125 @@
+# Test that guaranteed and burstable pods get the CPUs they require
+# when there are enough CPUs available.
+
+# pod0, fits in a core
+CPU=1 create guaranteed
+report allowed
+verify \
+    'len(cpus["pod0c0"]) == 1'
+
+# pod1, takes full core
+CPU=2 create guaranteed
+report allowed
+verify \
+    'len(cpus["pod0c0"]) == 1' \
+    'len(cpus["pod1c0"]) == 2' \
+    'len(cores["pod1c0"]) == 1' \
+    'len(set.union(cpus["pod0c0"], cpus["pod1c0"])) == 3'
```

Comment:
I understand that set theory and these 4 assertions imply that the intersection of cpus["pod0c0"] and cpus["pod1c0"] is empty, yet I would feel safer/prefer replacing the last assertion with an intersection check, because it is just much more natural considering the semantics of guaranteed pods with exclusive/full integer CPU requirements. And this applies to most of the subsequent checks to the same effect.

Reply:
Sure, I'll change the last ones to `set.intersection` and mark resolved when done.